package.json

```diff
@@ -1,50 +1,50 @@
 {
-  "name": "@socket.io/mongo-adapter",
-  "version": "0.3.0",
-  "description": "The Socket.IO MongoDB adapter, allowing to broadcast events between several Socket.IO servers",
-  "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:socketio/socket.io-mongo-adapter.git"
-  },
-  "files": [
-    "dist/"
-  ],
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
-  "scripts": {
-    "test": "npm run format:check && tsc && nyc mocha --require ts-node/register test/index.ts",
-    "format:check": "prettier --parser typescript --check 'lib/**/*.ts' 'test/**/*.ts'",
-    "format:fix": "prettier --parser typescript --write 'lib/**/*.ts' 'test/**/*.ts'",
-    "prepack": "tsc"
-  },
-  "dependencies": {
-    "debug": "~4.3.1",
-    "mongodb": "*"
-  },
-  "peerDependencies": {
-    "socket.io-adapter": "^2.5.2"
-  },
-  "devDependencies": {
-    "@types/expect.js": "^0.3.29",
-    "@types/mocha": "^8.2.1",
-    "@types/node": "^14.14.7",
-    "expect.js": "0.3.1",
-    "mocha": "^10.2.0",
-    "nyc": "^15.1.0",
-    "prettier": "^2.1.2",
-    "socket.io": "^4.6.1",
-    "socket.io-client": "^4.6.1",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.9.4"
-  },
-  "engines": {
-    "node": ">=10.0.0"
-  },
-  "keywords": [
-    "socket.io",
-    "mongodb",
-    "mongo",
-    "adapter"
-  ]
-}
+    "name": "@socket.io/mongo-adapter",
```

Comment:
It seems something in your environment is breaking the format of the `package.json` file, could you please check?

Reply:
Oh woops, seems vscode used four spaces as default.

I pushed again, but noticed you already made some fixes, so I guess we can close this PR then anyways 😇